cmd/rhobs/logs_test.go

```
@@ -128,6 +128,120 @@ func TestResolveLogsNamespace(t *testing.T) {
+		},
+		{
+			name:      "URL with query string bypass",
+			cellUrl:   "https://attacker.invalid/?x=.rhobs.api.openshift.com",
+			wantError: true,
+		},
+		{
+			name:      "URL with path bypass",
+			cellUrl:   "https://attacker.invalid/path.rhobs.api.openshift.com",
+			wantError: true,
+		},
+		{
```

golangci-lint (2.12.2)

[high] G101: Potential hardcoded credentials: Password in URL

(gosec)